app/src/main/java/com/andrew/tasky/agenda/presentation/screens/task_detail/TaskDetailViewModel.kt

```diff
@@ -84,12 +82,12 @@ class TaskDetailViewModel @Inject constructor(
         )
         viewModelScope.launch {
             withContext(NonCancellable) {
-                localRepository.upsert(agendaItem)
-                if (savedStateHandle.get <AgendaItem.Task("agendaItem")?.apiId == null) {
-                    // remoteRepository.createTask(agendaItem) TODO
-                } else {
-                    // remoteRepository.updateTask(agendaItem) TODO
-                }
+                // localRepository.upsert(agendaItem)
+                // if (savedStateHandle.get <AgendaItem.Task("agendaItem")?.id == null) {
+                //    // remoteRepository.createTask(agendaItem) TODO
+                // } else {
+                //    // remoteRepository.updateTask(agendaItem) TODO
+                // }
```

Comment:
Throw it away if you don't need it :) If you do need it again, you can use Git's commit history and compare the changes with this commit.